bin/check-configservice-rules.rb

```diff
@@ -0,0 +1,80 @@
+#! /usr/bin/env ruby
+#
+# check-configservice-rules
+#
+# DESCRIPTION:
+#   This plugin uses the AWS ConfigService API to check
+#   for rules compliance. CRITICAL for non-compliance,
+#   UNKNOWN for insufficient data.
+#
+# OUTPUT:
+#   plain-text
+#
+# PLATFORMS:
+#   Linux, Windows, Mac
+#
+# DEPENDENCIES:
+#   gem: aws-sdk
+#   gem: sensu-plugin
+#
+# USAGE:
+#  ./check-configservice-rules.rb -r {us-east-1|eu-west-1}
+#
+# NOTES:
+#   As of this writing, ConfigService rules are only available in us-east-1.
+#   All other region selections will return an AccessDeniedException
+#
+# LICENSE:
+#   Norm MacLennan <nmaclennan@cimpress.com>
+#   Released under the same terms as Sensu (the MIT license); see LICENSE
+#   for details.
+
+require 'sensu-plugin/check/cli'
+require 'sensu-plugins-aws'
+require 'aws-sdk'
+
+class CheckConfigServiceRules < Sensu::Plugin::Check::CLI
+  include Common
+
+  option :aws_region,
+         short: '-r AWS_REGION',
+         long: '--aws-region AWS_REGION',
+         description: 'The AWS region in which to check rules. Currently only available in us-east-1.',
+         default: 'us-east-1'
+
+  option :config_rules,
+         short: '-c rule1,rule2',
+         long: '--config-rules rule1,rule2',
+         description: 'A list of config rules to consider. Default is all rules.'
+
+  def aws_client(opts = {})
```

Comment:
I considered doing that (especially since I've got another PR following this one that does stubbing and testing in the same way) but in the interest of not going through all of the other files that include Common to ensure this doesn't break them in some way, I chickened out.

It doesn't seem likely that it *would* break anything, so I'd certainly be willing to file another PR with the explicit purpose of moving aws_client down into Common if you're cool with it.